Update the user and group to _puppet
Fix module search path to /var/puppetlabs

Index: lib/puppet/defaults.rb
--- lib/puppet/defaults.rb.orig
+++ lib/puppet/defaults.rb
@@ -57,7 +57,7 @@ module Puppet
       end
       path.join(File::PATH_SEPARATOR)
     else
-      '$codedir/modules:/opt/puppetlabs/puppet/modules'
+      '$codedir/modules:/var/puppetlabs/puppet/modules'
     end
   end
 
@@ -70,7 +70,7 @@ module Puppet
         nil
       end
     else
-      '/opt/puppetlabs/puppet/vendor_modules'
+      '/var/puppetlabs/puppet/vendor_modules'
     end
   end
 
@@ -1312,13 +1312,13 @@ EOT
 
   settings.define_settings(:server,
     :user => {
-      :default    => "puppet",
+      :default    => "_puppet",
       :desc       => "The user Puppet Server will run as. Used to ensure
       the agent side processes (agent, apply, etc) create files and
       directories readable by Puppet Server when necessary.",
     },
     :group => {
-      :default    => "puppet",
+      :default    => "_puppet",
       :desc       => "The group Puppet Server will run as. Used to ensure
       the agent side processes (agent, apply, etc) create files and
       directories readable by Puppet Server when necessary.",
